src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IReturnStatement.cs

```diff
@@ -421,6 +421,42 @@ int F(bool a)
             VerifyFlowGraphForTest<BlockSyntax>(compilation, expectedGraph);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void ReturnFlow_UnaryNegation()
+        {
+            var source = @"
+class C
+{
+    bool F(bool f)
+    /*<bind>*/{
+        return !f;
+    }/*</bind>*/
+}";
+
+            var compilation = CreateCompilation(source);
+
+            compilation.VerifyDiagnostics();
+
+            string expectedGraph = @"
+Block[B0] - Entry
+    Statements (0)
+    Next (Regular) Block[B1]
+Block[B1] - Block
+    Predecessors: [B0]
+    Statements (0)
+    Next (Return) Block[B2]
+        IUnaryOperation (UnaryOperatorKind.Not) (OperationKind.UnaryOperator, Type: System.Boolean) (Syntax: '!f')
```

Comment:
```suggestion
        IUnaryOperation (UnaryOperatorKind.Not) (OperationKind.Unary, Type: System.Boolean) (Syntax: '!f')
```

Looks like you need to switch the OperationKind text being used here for the standardized one, this was changed in a recent commit. It should be fine with the suggested change.